pkgs/servers/tailscale/default.nix

```diff
@@ -24,6 +25,16 @@ buildGoModule {
     rev = "v${version}";
     hash = "sha256-1Yt8W/UanAghaElGiD+z7BKeV/Ge+OElA+B9yBnu3vw=";
   };
+
+  patches = [
+    # Fix "tailscale ssh" when built with ts_include_cli tag
+    # https://github.com/tailscale/tailscale/pull/12109
```

Comment:
I was wondering if we should wait for the review in the upstream and then include this patch :)

Reply:
This is a none security relevant change and we understand the underlying issue pretty well, so we don't necessarily need the upstream review.